docs-website/docs/overview/sdk.md

```diff
@@ -2,3 +2,92 @@
 sidebar_position: 3
 title: SDK
 ---
+
+# typesafe-i18n
+An opinionated, fully type-safe, lightweight localization library for TypeScript and JavaScript projects with no external dependencies.
+
+## Installation
+### 1.1. Install typesafe-i18n, the importer and concurrently
+
+
+```bash
+npm i typesafe-i18n
+npm i @inlang/typesafe-i18n-importer
+npm i concurrently --save-dev
+```
+
+### 1.2. Create the .typesafe-i18n.json config file
+
+- `adapter` specifies that the generates i18n files should be react compatible.
```

Comment:
What does the adapter do? It's purpose is surely not to be react compatible, especially when the config file below uses svelte.